process/block/shardblock.go

```diff
@@ -1823,3 +1841,10 @@ func (sp *shardProcessor) getBootstrapHeadersInfo(
 
 	return lastSelfNotarizedHeaders
 }
+
+func (sp *shardProcessor) removeStartOfEpochBlockDataFromPools(
+	headerHandler data.HeaderHandler,
```

Comment:
add _ instead of variable name for those which are not used.

Reply:
Done